engine-tests/src/test/java/org/terasology/math/ChunkMathTest.java

```diff
@@ -93,4 +93,14 @@ public void testCalcChunkPos() {
         assertTrue(ChunkMath.calcChunkPos(10, 10, 10, temp).equals(0, 0, 0));
     }
 
+
+    @Test
+    public void testFloatingPointCalcChunkPos() {
```

Comment:
That's some exactly-half, over-half, under-half, that's good.

Throw in at least one test with some negative coordinates, so we can tell if it's doing round-towards-zero or round-towards `-infinity`. What should that be... I guess x -0.1 is in chunk -1, along with x -63.9

x: -64 and -64.1 are chunk -2? bah, the fencepost errors are messing with me. I keep confusing myself, I should draw more diagrams.